docs/user/aws/install_upi.md

```diff
@@ -0,0 +1,432 @@
+# Install: User Provided Infrastructure (UPI)
+
+The steps for performing a UPI-based install are outlined here. Several [CloudFormation][cloudformation] templates are
+provided to assist in completing these steps or to help model your own.
+
+## Create Ignition Configs
+
+The machines will be started manually. Therefore, it is required to generate the bootstrap and machine Ignition configs
+and store them for later steps.
+
+```console
+$ openshift-install-linux-amd64 create ignition-configs
+? SSH Public Key /home/user_id/.ssh/id_rsa.pub
+? Platform aws
+? Region us-east-1
+? Base Domain example.com
+? Cluster Name openshift
+? Pull Secret [? for help]
+```
+
+After running the command, several files will be available in the directory.
+
+```console
+$ tree
+   .
```

Comment:
nit: these `tree` output lines should not be indented.

Reply:
Fixed.